src/java/za/co/mip/ablduck/utilities/templates.properties

```diff
@@ -0,0 +1,98 @@
+CLASSBODY=\
+<div>\
+	{0}\
+	<div class="doc-contents">\
+		{1}\
+	</div>\
+	<div class="members">\
+		{2}\
+		{3}\
+	</div>\
+</div>
+SIDEBAR=\
+<pre class="hierarchy">\
+	{0}\
+	{1}\
+	{2}\
+</pre>
+AUTHOR=\
+<h4>Author</h4>\
+<div class="dependency">{0}</div>
+HIERARCHY=\
+<h4>Hierarchy</h4>\
+{0}
+HIERARCHY_ITEM=\
+<div class="subclass {0}">\
+{1}\
+{2}\
+</div>
+SUBCLASSES=\
+<h4>Subclasses</h4>\
+{0}
+SUBCLASS_ITEM=<div class="dependency">{0}</div>
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename key "SUBCLASS_ITEM" to match the regular expression: ^[A-Za-z][.A-Za-z0-9]*$ [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar.riverside-software.fr/coding_rules#rule_key=jproperties%3Akey-naming-convention)